samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the :ref:<lib_zigbee_shell> library for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as coordinator, adding devices to the network with the install codes, setting the extended PAN ID.
+* Benchmarking - measuring time needed for a message to travel from one node to another.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
+
+You can use two or more of the development kits listed above and mix different development kits.
+
+Additionally, you can use this sample with any other :ref:`Zigbee <ug_zigbee>` sample application for testing the communication or other sample-specific functionalities.
+
+Overview
+********
+
+The Zigbee shell sample takes the Zigbee Router role and implements two clusters: Basic and Identify.
+The Basic cluster provides attributes and commands for determining basic information about the node.
+The Identify cluster allows to put the device into the identification mode, which provides a way to locate the device.
+The device also includes all Zigbee shell commands, that allow for discovering, controlling and testing other Zigbee devices.
+
+Available CLI interfaces
+************************
+
+You can run the Zigbee shell commands after connecting and configuring any of the :ref:<lib_zigbee_shell> supported backends for testing.
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+For more information, see :ref:`gs_testing`.
+
+The sample demonstrates the usage of commands from the :ref:`zigbee_cli_reference`.
+Zigbee CLI is integrated into the system shell that is accessible over the available interfaces.
```

Comment:
`Zigbee CLI` or `Zigbee shell`? Think we need to decide if we should get rid of `Zigbee CLI`